experiment/run_experiment.py

```diff
@@ -239,6 +239,11 @@ def start_experiment(  # pylint: disable=too-many-arguments
     config['no_dictionaries'] = no_dictionaries
     config['oss_fuzz_corpus'] = oss_fuzz_corpus
     config['description'] = description
+    config['runner_machine_type'] = config.get('runner_machine_type',
+                                               'n1-standard-1')
+    config['runner_num_cpu_cores'] = config.get('runner_num_cpu_cores', 1)
+    # Note this is only used if runner_machine_type is None.
+    config['runner_memory'] = config.get('runner_memory', '12GB')
```

Comment:
maybe move 12 gb to some constants file

Reply:
12 GB is just useful for klee. i'll mention that in a comment. I don't really think it should be a constant.
I'll probably just make this a comment.